### DISCUSSION TITLE
Why the v2beta search method doesn't return resourceOwner in result set?

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8035] [Q&A] [answered]
The problem is I am using [v2beta search API] (https://zitadel.com/docs/apis/resources/user_service/user-service-list-users) to find a user and afterward, a need to obtain its metadata. The thing is that v2beta API doesn't provide a method for user metadata retrieving, thus I have to use v1 API, which requires me to provide organization ID, which I don't have in v2beta search API response. How can I get user's org ID without performing unnecessary requests? Thank you.

**Comment by hifabienne** [comment]
@stebenz @eliobischof can you help here?

**Reply by ashep** [reply, marked as answer]
Seems fixed for now.

https://zitadel.com/docs/apis/resources/user_service_v2/user-service-list-users
